laravel/resources/views/layouts/head.blade.php

```diff
@@ -0,0 +1,2 @@
+<!-- CSS and JS -->
+@vite(['resources/sass/app.scss', 'resources/js/app.js'])
```

Comment:
[SUGGESTION]

To reduce code complexity, I would move this directly into the `layout.blade.php`.

It doesn't make sense to outsource such simple code to an extra file.